NEWS.txt

```diff
@@ -3,6 +3,34 @@ Release Notes
 =============
 
 
+Theano 0.9.0rc3 (6th of March, 2017)
+====================================
+
+This release extends the 0.9.0rc2 and announces the upcoming final release 0.9.
+
+Highlights (since 0.9.0rc2):
+ - Graph clean up and faster compilation
+ - Removed warp-synchronous programming
+ - New Theano flag conv.assert_shape
+ - Fix precision in cuDNN
```

Comment:
Remove, it don't explain the problem and people won't understand what this mean. I would consider that a crash fixes.